ccmlib/scylla_node.py

```diff
@@ -536,51 +537,69 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
         jvm_args = jvm_args + ['--collectd-hostname',
                                '%s.%s' % (socket.gethostname(), self.name)]
 
-        # Let's add jvm_args and the translated args
-
-        args = [launch_bin, '--options-file', options_file, '--log-to-stdout', '1'] + jvm_args + translated_args
+        args = [launch_bin, '--options-file', options_file, '--log-to-stdout', '1']
+
+        # holds all extended args except --smp and --memory
+        # that are handled separately
+        ext_args = OrderedDict()
+
+        MB = 1024*1024
```

Comment:
nit, might want to add spaces around operator.

Reply:
added in v2